snuba/datasets/errors_processor.py

```diff
@@ -100,7 +99,7 @@ def extract_contexts_custom(
         contexts: Mapping[str, Any],
         metadata: KafkaMessageMetadata,
     ) -> None:
-        output["_contexts_flattened"] = ""
+        pass
```

Comment:
I'd just remove the method from the parent class. None is supposed to use this now.